src/win32_window.c

```diff
@@ -925,9 +931,38 @@ static LRESULT CALLBACK windowProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM l
 
             data = _glfw.win32.rawInput;
             if (data->data.mouse.usFlags & MOUSE_MOVE_ABSOLUTE)
-            {
-                dx = data->data.mouse.lLastX - window->win32.lastCursorPosX;
-                dy = data->data.mouse.lLastY - window->win32.lastCursorPosY;
+            {   
+                if (_glfw.win32.isRemoteSession)
+                {
+                    //Remote Desktop Mode...
+                    // As per https://github.com/Microsoft/DirectXTK/commit/ef56b63f3739381e451f7a5a5bd2c9779d2a7555
+                    // MOUSE_MOVE_ABSOLUTE is a range from 0 through 65535, based on the screen size.
+                    // As far as I can tell, absolute mode only occurs over RDP though.
+                    width = GetSystemMetrics((data->data.mouse.usFlags & MOUSE_VIRTUAL_DESKTOP) ? SM_CXVIRTUALSCREEN : SM_CXSCREEN);
+                    height = GetSystemMetrics((data->data.mouse.usFlags & MOUSE_VIRTUAL_DESKTOP) ? SM_CYVIRTUALSCREEN : SM_CYSCREEN);
+
+                    pos.x = (int)((data->data.mouse.lLastX / 65535.0f) * width);
+                    pos.y = (int)((data->data.mouse.lLastY / 65535.0f) * height);
+                    ScreenToClient(window->win32.handle, &pos);
+
+
+                    _glfwGetWindowSizeWin32(window, &window_width, &window_height);
+
+                    // One other unfortunate thing is that re-centering the cursor will still fire an
+                    // input event; assume that any motion to the center is our re-centering and ignore it
+                    if (pos.x == window_width / 2 && pos.y == window_height / 2)
+                        break;
```

Comment:
I think the point is to prevent raising twice the cursorPos callbacks when the cursor is recentered by code (in _glfwPollEventsWin32). I tested it over Remote Desktop and i hit the break on line 954.
![image](https://github.com/glfw/glfw/assets/81109165/f0700bd4-789c-40f5-9127-5f3f32285662)

You are right that the resulting is dx and dy are zero. If it's ok to raise the event twice when the mouse is moved, i can remove the check, but it's seems to me not a good idea? Maybe there's something else i don't undertand?

Reply:
Oh nevermind, after further check , I found a check in _glfwInputCursorPos that prevent raising cursorPos if the last virtualcursorX and Y and identical. That shoud prevent the cursorPos trice.